lib/build.gradle

```diff
@@ -41,7 +41,7 @@ compileJava {
 }
 
 dependencies {
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.5.1'
```

Comment:
I think `2.10.5.1` is already bug-free according to https://nvd.nist.gov/vuln/detail/CVE-2020-25649
Please note, the known affected version is "Up to (excluding) 2.10.5.1"

Reply:
Hi Luke,
Can you please refer below links:

1. https://bugzilla.redhat.com/show_bug.cgi?id=1887664
2. https://github.com/FasterXML/jackson-databind/commit/e588f0af61b18576779ffb95a2a689a5eb1f9d15

Reply:
Hi,
Yeah, I guess 2.11.0 might be a good one. Unfortunately, I am not from Auth0. Just want them to fix this issue as soon as possible~